paper-typeahead-input.html

```diff
@@ -313,7 +340,7 @@
         }
       });
       var input = Polymer.dom(this.root).querySelector('paper-input');
-      input.$.input.autocomplete = 'address-level4';
+      input.$.input.autocomplete = this.autocomplete;
```

Comment:
To prevent autofilling of Chrome, autocomplete="Off" is not a reliable solution. Chrome Browser Ignoring AutoComplete=Off. See discussion here http://stackoverflow.com/questions/12374442/chrome-browser-ignoring-autocomplete-off.   And plus, the expected behavior of paper-typeahead-input is to provide its own suggest, so I don't see why we need to expose this autocomplete property of inner input as the property of paper-typeahead-input.